panda/metalibs/panda/CMakeLists.txt

```diff
@@ -5,6 +5,10 @@ set(PANDA_LINK_TARGETS
   p3pnmimage p3pnmimagetypes p3pstatclient p3putil p3recorder p3text p3tform
 )
 
+if(HAVE_TINYDISPLAY)
+  list(APPEND PANDA_LINK_TARGETS p3tinydisplay)
+endif()
```

Comment:
I don't think this should be linked into libpanda.  tinydisplay doesn't have to be built as a metalib.

Reply:
but right now the current cmake changes in master for tinydisplay don't work
for static build 
```
CMake Error: install(EXPORT "Framework" ...) includes target "p3framework" which requires target "p3tinydisplay" that is not in any export set.
CMake Error in panda/src/framework/CMakeLists.txt:
  export called with target "p3framework" which requires target
  "p3tinydisplay" that is not in any export set.

```

Reply:
If you need me to investigate this please open an issue for it describing how I can repro it